### PR TITLE
Add Panasonic icon

### LINKED ISSUE #5233: Request: Panasonic
**Brand Name:Panasonic**
**Website:https://www.panasonic.com/**
**Alexa rank:4,371**
  

**Official resources for icon and color:**

### PR DESCRIPTION
**Pull request by PratyushJayachandran**
panasonic

**Issue:** closes #5233 

**Similarweb rank:** 4,502. See https://www.similarweb.com/website/panasonic.com

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
SVG: Taken from website header at https://www.panasonic.com
Color: `#0049ab` Taken from website header and footer.

### CODE REVIEW

**Review by adamrusted — CHANGES_REQUESTED** [review]
I'm seeing some [slight differences](https://github.com/simple-icons/simple-icons/assets/1830867/63960a20-26d3-42fc-a70e-27c73198380d) (ignoring where their own SVG is broken!) between your PR in black and the source icon in red. I'd also steer away from having `www.na.` in the URL, and instead default it to just `www.`.

**Comment by PratyushJayachandran** [comment]
> I'm seeing some [slight differences](https://github.com/simple-icons/simple-icons/assets/1830867/63960a20-26d3-42fc-a70e-27c73198380d) (ignoring where their own SVG is broken!) between your PR in black and the source icon in red. I'd also steer away from having `www.na.` in the URL, and instead default it to just `www.`.

This is one of the more you look at it cases. Anyway I tried to fix it. Please review.

**Review by adamrusted — CHANGES_REQUESTED** [review]
I'm still spotting a [slight difference](https://github.com/simple-icons/simple-icons/assets/1830867/5ed2c4e0-090e-4fd7-a558-1fb221182c7d) I'm afraid @PratyushJayachandran - might need to up the precision to 4 to fix it.

**Comment by PratyushJayachandran** [comment]
> I'm still spotting a [slight difference](https://github.com/simple-icons/simple-icons/assets/1830867/5ed2c4e0-090e-4fd7-a558-1fb221182c7d) I'm afraid @PratyushJayachandran - might need to up the precision to 4 to fix it.

[It is not perfect](https://github.com/simple-icons/simple-icons/assets/35000807/6fea91d0-efe5-443d-89bc-19b0c06af191) but it is better than before.

**Review by adamrusted — CHANGES_REQUESTED** [review]
Unfortunately still seeing [quite a difference](https://github.com/simple-icons/simple-icons/assets/1830867/ce8826d7-276a-4ced-b9b8-6e05b2968144) between your PR and [this file](https://www.panasonic.com/content/dam/Panasonic/plogo.svg) from the header.

**Comment by PratyushJayachandran** [comment]
> Unfortunately still seeing [quite a difference](https://github.com/simple-icons/simple-icons/assets/1830867/ce8826d7-276a-4ced-b9b8-6e05b2968144) between your PR and [this file](https://www.panasonic.com/content/dam/Panasonic/plogo.svg) from the header.

I hope it is fixed now. I might have been using the wrong source. I took your source, updated JSON with it and re-did all the process with precision 4. I didn't touch any point or try clean-ups
panasonic

**Comment by Wissididom** [comment]
I don't know it it is important but I think there is a typo in the PR title
```diff
- Add Pansonic icon
+ Add Panasonic icon
```

**Comment by PratyushJayachandran** [comment]
> I don't know if it is important but I think there is a typo in the PR title
> 
> ```diff
> - Add Pansonic icon
> + Add Panasonic icon
> ```

Thanks for pointing it out

File: _data/simple-icons.json
Comment (adamrusted) [inline _data/simple-icons.json:9026]:
```suggestion
            "source": "https://www.panasonic.com",
```
Although we're using a specific SVG, it's available in the website header. No need to link directly to the file.

File: icons/panasonic.svg
Comment (adamrusted) [inline icons/panasonic.svg:1]: Path itself now matches up perfectly. If you're able to resolve the imperfections without altering the shape, then please do. Otherwise, I'm happy to merge this in - unless another @simple-icons/icon-review maintainer is willing to take a look?

**Review by adamrusted — CHANGES_REQUESTED** [review]

File: icons/panasonic.svg
Comment (adamrusted) [reply icons/panasonic.svg:1]: If you can remove the discrepencies (like the weird chunks missing from letters) then go for it. I'm only anti-edit in as much as the main shape needs to remain the same. SVG optimisations are fine.

Comment (PratyushJayachandran) [reply icons/panasonic.svg:1]: I have cleaned up `P a o`
Precision 4 and checked from my side.

**Review by adamrusted — APPROVED** [review]
LGTM! Thanks @PratyushJayachandran 🎉

**Pull request merged**
